tests/general/InvalidLinkBearTest.py

```diff
@@ -154,10 +153,7 @@ def test_run(self):
         http://httpbin.org/status/404
         http://httpbin.org/status/410
         http://httpbin.org/status/500
-        http://httpbin.org/status/503
-        http://www.notexample.com/404
-        http://exampe.com/404
-        http://example.co.in/404"""
+        http://httpbin.org/status/503"""
```

Comment:
If the blank line is a problem , do https://github.com/coala/coala/issues/4227 first , so you can use `check_line_result_count` in this test method, and it can ignore blank lines.